text/ps-signatures/README.md

```diff
@@ -0,0 +1,135 @@
+- Feature Name: ps-signatures
+- Start Date: 2019-08-08
+- RFC PR: (leave this empty)
+- Ursa Issue: (leave this empty)
+- Version: 0.1
+
+# Summary
+[summary]: #summary
+
+Pointcheval-Sanders signature scheme from the paper [Short Randomizable signatures](https://eprint.iacr.org/2015/525) with protocol for proof of knowledge of signature. A user can get signature on known or committed messages from the signer and then prove knowledge of signature and optionally reveal some messages under the signature. Implementation in this [PR](https://github.com/hyperledger/ursa/pull/46). This feature depends on the [PR](https://github.com/hyperledger/ursa-rfcs/pull/10) for RFC for proof of knowledge of committed messages in a commitment. 
+
+# Motivation
+[motivation]: #motivation
+
+Anonymous credentials consists of issuer's signature over credential attributes with the property that it is efficient to prove knowledge of signature without revealing the signature itself. Additionally, it should be possible to reveal some of the attributes of the credential to the verifier. Anonymous credentials are needed in Hyperledger Aries.
```

Comment:
I think that the motivation was Aries wants to offer <usage x> which requires threshold signatures. Ursa does not offer a threshold sig feature. This PS scheme is adding a feature which is not otherwise available in the project.

Reply:
Regardless of threshold signatures, Zmix intended to support PS sig as an alternate scheme for BBS+.

Reply:
I don't recall a list of intended features in z-mix. Do you have a link? I looked in the project proposal and the archived lab repo.
What would be the motivation for an alternative for BBS+?

Reply:
The motivation was applicability in threshold issuance. The [Zmix RFC specifies](https://github.com/hyperledger/ursa-rfcs/blob/c75c55d0673a9e54df1c7d6479cbdd1d909f6bed/0000-zmix/README.md#statements) that PS should be supported but the decision was made long before the lab project. Now one might ask the reason for supporting BBS+ and i think the answer is that the assumptions made are popular whereas the assumption (LRSW) made in PS is unpopular.